nodejs/dist/uws.js

```diff
@@ -25,17 +29,22 @@ class Socket {
      * @param {Function} f
      */
     on(eventName, f) {
+        if (typeof f !== 'function') {
+            throw new TypeError('Event listener must be a function');
+        }
```

Comment:
I don't like this check though. JavaScript is 100% type-unsafe and if they call us with a space ship that's their problem.